### PR TITLE
update interim patch

### PR DESCRIPTION
**Pull request by J-shang**
### Checklist ###
  - [ ] test case
  - [ ] doc

### CODE REVIEW

**Review by ultmaster — COMMENTED** [review]
Maybe this patch is no longer needed?

**Comment by J-shang** [comment]
> Maybe this patch is no longer needed?

I will try if the `torchvision` patch is needed.
This pr is for `setuptools.distutils` has some issues in the latest version and will cause `torch.utils.tensorboard` break.

https://msrasrg.visualstudio.com/NNIOpenSource/_build/results?buildId=30308&view=logs&j=597b6a97-1552-58b1-ca74-8e543bcf8377&t=35dc6439-9fc2-5498-5aa9-345bd49c2643

File: interim_patch.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# fix setuptools.distutils import in torch
```
Comment (ultmaster) [inline interim_patch.py:1]: Can we move this file to `tests/vso_tools`?

Comment (J-shang) [reply interim_patch.py]: moved

**Review by ultmaster — APPROVED** [review]

**Review by liuzhe-lz — APPROVED** [review]

**Pull request merged**